packages/timeline-state-resolver/src/devices/deviceContainer.ts

```diff
@@ -17,11 +22,12 @@ export class DeviceContainer<TOptions extends DeviceOptionsBase<any>> {
 	public onChildClose: () => void | undefined
 	private _instanceId = -1
 	private _startTime = -1
-	private _onEventListener: { stop: () => void } | undefined
+	private _onEventListeners: { stop: () => void }[] | undefined
```

Comment:
I know this is a cherry-pick but there is no need for this to be `| undefined` when it is an array. Just have an empty array.

Reply:
Has been updated.